toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -44,25 +44,13 @@ type SystemConfig struct {
 // GetRootPartitionSetting returns a pointer to the partition setting describing the disk which
 // will be mounted at "/", or nil if no partition is found
 func (s *SystemConfig) GetRootPartitionSetting() (rootPartitionSetting *PartitionSetting) {
-	for i, p := range s.PartitionSettings {
-		if p.MountPoint == "/" {
-			// We want to reference the actual object in the slice
-			return &s.PartitionSettings[i]
-		}
-	}
-	return nil
+	return FindRootPartitionSetting(s.PartitionSettings)
```

Comment:
Can we do something like `return FindMountpointPartitionSetting(s.PartitionSettings, "/")` and just drop the function completely?

Reply:
nvm, see why we might want to keep these